superset/utils/log.py

```diff
@@ -168,6 +168,10 @@ def log(  # pylint: disable=too-many-locals
             )
             logs.append(log)
 
-        sesh = current_app.appbuilder.get_session
-        sesh.bulk_save_objects(logs)
-        sesh.commit()
+        try:
```

Comment:
I hit this interesting bug where if the sqlalalchemy session is left in a bad state, this can raise. I feel like the right/easy thing to do here is to silence this. Failing at logging an analytic event should never fail the app as hard as this did.